vendor/github.com/vmware/go-nfs-client/nfs/nfs.go

```diff
@@ -222,7 +264,10 @@ func dialService(addr string, port int) (*rpc.Client, error) {
 				Port: p,
 			}
 
-			client, err = rpc.DialTCP("tcp", ldr, fmt.Sprintf("%s:%d", addr, port))
+			raddr := fmt.Sprintf("%s:%d", addr, port)
+			util.Debugf("Connecting to %s", raddr)
+
+			client, err = rpc.DialTCP("tcp", ldr, raddr)
```

Comment:
We should probably make a constant at some point for `"tcp"` since it is used a lot. I am surprised that the `rpc` does not already have some thing like this.